test/integration/sequelize/deferrable.test.js

```diff
@@ -6,7 +6,7 @@ const expect = chai.expect;
 const Support = require('../support');
 const Sequelize = require('@sequelize/core');
 
-if (!Support.sequelize.dialect.supports.deferrableConstraints) {
+if (!Support.sequelize.dialect.supports.deferrableConstraints || Support.getTestDialect() === 'yugabytedb') { // Doesn't support Deferrable Constraints properly, refer- https://github.com/yugabyte/yugabyte-db/issues/9288
```

Comment:
Then you should disable it in your dialect's flag instead

Reply:
Actually we support deferrable constraints, but there are cases mentioned [here](https://github.com/yugabyte/yugabyte-db/issues/9288) due to which we want to skip some test here, will skip those only instead of whole `deferrable.test.js`.

Reply:
I see, thanks :)